README.md

```diff
@@ -126,6 +126,21 @@ top-level make command line. These options, such as *PLATFORM_<xyz>* or
 *docs/platform/<platform_name>.md* files and
 *docs/firmware/<firmware_name>.md* files.
 
+Providing different payloads to OpenSBI Firmware
+------------------------------------------------
+OpenSBI firmware can accept various payloads as an compile time options.
+Typically, these payload refer to the next stage boot loader (e.g. U-Boot)
+or supervisor operating system images (e.g. Linux). By default, OpenSBI
+firmware includes a test payload in absence of FW_PAYLOAD_PATH argument.
+
+To provide a payload during compile time
+```
+make PLATFORM=<platform_subdir> FW_PAYLOAD_PATH=<payload path>
+```
+The instructions to build each payload is different and the details can
+be found in the
+*docs/firmware/payload_<payload_name>.md* files.
+
```

Comment:
Move this section from top-level README.md to docs/firmware/fw.md

You can call section in docs/firmware/fw.md as "Firmware Payloads"

Reply:
done.